scripts/artifacts-building/containers/build-process.sh

```diff
@@ -123,6 +123,10 @@ ansible_tag_filter "openstack-ansible containers/artifact-build-chroot.yml -e ro
 ansible_tag_filter "openstack-ansible containers/artifact-build-chroot.yml -e role_name=repo_server -v" "install" "config"
 ansible_tag_filter "openstack-ansible containers/artifact-build-chroot.yml -e role_name=rsyslog_server -v" "install" "config"
 
+# test one container build contents
```

Comment:
OK, it will mean that this test takes ages - but I guess it takes less time than a deploy so it should be OK. We can also try to optimise the build later.